README.md

```diff
@@ -25,3 +25,10 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+Recurrence relation: T(n) = 3T(n/3) + 1
```

Comment:
How many operations are you performing to partition the array and combine the partial results?